### PR TITLE
fix more presets

### PR DESCRIPTION
**Pull request by krzyzacy**
that find/replace seems did not work super great :-(
/shrug
/assign @BenTheElder

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -20726,18 +20742,18 @@ periodics:
...
+        -label:do-not-merge/work-in-progress
+        label:lgtm
+        label:approved
         status:failure
-        -labels:needs-rebase
-        -labels:needs-ok-to-test
-        -labels:"cncf-cla: no"
+        -label:needs-rebase
+        -label:needs-ok-to-test
+        -label:"cncf-cla: no"
```
Comment (krzyzacy) [inline prow/config.yaml]: (also shrug for those - not sure if there are more instances :-( )

Comment (BenTheElder) [reply prow/config.yaml]: what's going on here?

Comment (BenTheElder) [reply prow/config.yaml]: oh :(((

Comment (krzyzacy) [reply prow/config.yaml]: hummm, assume you did something like `s/-label/-labels` to this file? I was diff the job yaml for this job and this was changed

Comment (BenTheElder) [reply prow/config.yaml]: yeah :/
I'll go back over the previous diff to check for more of these.

File: prow/config.yaml [hunk 2]
```diff
@@ -15351,6 +15360,9 @@ periodics:
       - mountPath: /etc/ssh-key-secret
         name: ssh
         readOnly: true
+      - mountPath: /etc/service-account
+        name: service
+        readOnly: true
```
Comment (krzyzacy) [inline prow/config.yaml]: (this is for the regional job was actually using a different service account, need to make a different preset for them)

**Pull request merged**